### ISSUE TITLE
CONCEPT: Glucose Page

### ISSUE THREAD

**Issue by angelsix** [angelsix/blueberry#7]
We should agree on and define the structure of the Glucose page. It's purpose is to enter your Glucose readings, either manually, or via automated BT devices. 

I think the page should default to showing the dial to enter your current reading, and have a time, which defaults to now.

On top of that I think perhaps a list of connected BT glucose readers that we are already paired to, and for those not yet paired, a button to Connect/Pair with a new device. That way syncing with existing BT devices is not two taps away, its on the main page, and you just open the glucose page and click Sync on the device name. We could even automate this after but for now lets keep it simple.

Ideas/thoughts? 

Here is some concept UI ignore the specific text and buttons we can hard-define that once we are happy how this will work

**Comment by DicksHalfwayInn** [comment]
Yes, I agree Luke... pairing to a device can be in both the settings page and on the glucose reading entry page.  Pairing will probably happen automatically when you are within range of the bluetooth device.

idea 1:
- have a notification screen pop up on the phone when a device is connected, with an option to open the app to the glucose level input screen.  This might be a good way of reminding the user to enter a reading.

idea 2:  for manual entry of the glucose reading we could have a rolling dial control?  Sorry, I am pretty sure it must be a type of control, but I don't know what it is called.  It would work like the spinner on a slot machine with the little window showing the entry and the user can swipe up and down with their fingers to get to the correct reading and then a big Enter button when they have set the wheel to their reading.  I have attached a picture to show what I am talking about.  the spinner would show... 
0.0
0.1
0.2
…
19.5
19.6
etc.

idea 3:  There should be a notification system to remind people to take readings... the user should be able to enter an interval of time when they would like to take readings... or set the times directly.. 7:00 am, 9:00am 12:00pm, 4:00pm, etc... so that they can receive notification reminders to take their readings... this could also have a link on the glucose reading entry page
